world/console.cpp

```diff
@@ -406,6 +406,36 @@ void ConsoleGMSay(
 	zoneserver_list.SendChannelMessage(tmpname, 0, 11, 0, JoinString(args, " ").c_str());
 }
 
+/**
+ * @param connection
+ * @param command
+ * @param args
+ */
+void ConsoleGuildSay(
+	EQ::Net::ConsoleServerConnection *connection,
+	const std::string &command,
+	const std::vector<std::string> &args
+)
+{
+	if (args.size() < 1) {
+		return;
+	}
+
+	auto from = args[0];
+	auto guild_id = std::stoul(args[1]);
```

Comment:
Since `stoul` can throw you might want to just use `strtoul` here and abort if 0 to avoid crashing world with a bad input